### PR TITLE
unify workers

### PR DESCRIPTION
**Pull request by matanper**

### CODE REVIEW

File: backend/deepchecks_monitoring/api/v1/model.py [hunk 1]
```diff
@@ -52,6 +51,7 @@
 from deepchecks_monitoring.schema_models.monitor import Monitor, round_off_datetime
 from deepchecks_monitoring.utils import auth
 
+from ...public_models.task import delete_monitor_tasks
```
Comment (JKL98ISR) [inline backend/deepchecks_monitoring/api/v1/model.py:54]: ...

File: backend/deepchecks_monitoring/bgtasks/scheduler.py [hunk 2]
```diff
@@ -27,12 +27,12 @@
 from sqlalchemy.ext.asyncio import AsyncEngine, AsyncSession, create_async_engine
 from sqlalchemy.orm import joinedload, load_only, sessionmaker
 
-from deepchecks_monitoring import __version__, config
-from deepchecks_monitoring.bgtasks.core import Task
+from deepchecks_monitoring import config
+from deepchecks_monitoring.bgtasks.alert_task import QUEUE_NAME
```
Comment (JKL98ISR) [inline backend/deepchecks_monitoring/bgtasks/scheduler.py:31]: maybe use it the way I used? from the class

File: backend/deepchecks_monitoring/bgtasks/alert_task.py [hunk 3]
```diff
@@ -14,28 +14,84 @@
...
+from deepchecks_monitoring.public_models import Organization
+from deepchecks_monitoring.public_models.task import BackgroundWorker, Task
 from deepchecks_monitoring.resources import ResourcesProvider
 from deepchecks_monitoring.schema_models.alert import Alert
 from deepchecks_monitoring.schema_models.alert_rule import AlertRule, Condition
 from deepchecks_monitoring.schema_models.model_version import ModelVersion
 from deepchecks_monitoring.schema_models.monitor import Frequency, Monitor, as_pendulum_datetime
+from deepchecks_monitoring.utils import database
 
-__all__ = ["execute_monitor"]
```
Comment (JKL98ISR) [inline backend/deepchecks_monitoring/bgtasks/alert_task.py:35]: have __all__ = AlertsTask

File: backend/deepchecks_monitoring/schema_models/model.py
Comment (matanper) [inline backend/deepchecks_monitoring/schema_models/model.py:1]: @JKL98ISR

**Review by ItayGabbay — APPROVED** [review]

**Pull request merged**